sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -19,8 +19,7 @@ internal class RequestActivityPolicy : HttpPipelinePolicy
         private const string RequestIdHeaderName = "Request-Id";
 
         private static readonly DiagnosticListener s_diagnosticSource = new DiagnosticListener("Azure.Core");
-#if NETCOREAPP2_1
-        private static readonly object? s_activitySource = ActivityExtensions.CreateActivitySource("Azure.Core.Http");
```

Comment:
it might be easier to just set s_activitySource to null on netcoreapp2.1 and remove #if ... from the rest of this file